extensions/wikia/UserLogin/js/UserBaseAjaxForm.js

```diff
@@ -62,19 +75,34 @@
 
 	/**
 	 * Make the call to the back end to log the user in via ajax
+	 * @abstract
 	 */
-	UserBaseAjaxForm.prototype.ajaxLogin = function () {
-		$.nirvana.postJson(
-			'UserLoginSpecial',
-			'login',
-			{
-				loginToken: this.loginToken,
-				username: this.inputs.username.val(),
-				password: this.inputs.password.val(),
-				keeploggedin: this.inputs.keeploggedin.is(':checked')
-			},
-			this.submitLoginHandler.bind(this)
-		);
+	UserBaseAjaxForm.prototype.ajaxLogin = function () {};
```

Comment:
In UserLogin.js there are two another empty functions like this one.
Maybe we could define it in one place for this whole extension and use only reference to it. I mean something similar like jQuery.noop().
Three uses like in this extension is not much, but it is always a good practice to reuse a functions which does something (or nothing like in here). Ideally it would be to have one place for our whole code for utilities like this.